app/services/media_storage/azure_adapter.rb

```diff
@@ -35,7 +35,7 @@ def get_path(path, opts={})
       if opts[:private]
         azure_path = StoredPath.media_path(path)
         signer.signed_uri(
-          client.generate_uri("#{private_container}/#{azure_path}"),
+          client.generate_uri(File.join(private_container, azure_path)),
```

Comment:
Not blocking, i'd normally pull this line out to an explicit variable assignment
``` ruby
        azure_container_path = File.join(private_container, azure_path)
#.....
          client.generate_uri(azure_container_path),
```